index.html

```diff
@@ -975,6 +980,86 @@ <h2>Methods</h2>
             selected candidate pair =].
           </p>
         </dd>
+        <dt>
+          <dfn>removeCandidatePairs</dfn>
+        </dt>
+        <dd>
+          <p>
+            The {{removeCandidatePairs}} method removes the provided candidate pairs, making them unavailable to use
+            for transport. The [= ICE agent =] may free up candidates once all the associated candidate pairs have been removed.
```

Comment:
I'm not sure the last statement is always correct.  Won't a web app be able to keep a local candidate alive even if there are no candidate pairs at the moment?

Reply:
The current extent of the (merged) API only allows the application to keep candidate pairs alive, not the candidates themselves. We can revisit the statement once the scope expands to candidate manipulation as well.

Reply:
A "pair" is just a dictionary containing two [RTCIceCandidate](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate) objects, which in turn are [not backed by any network resources](https://w3c.github.io/webrtc-pc/#dfn-creating-an-rtcicecandidate). They can be created out of thin air, and thus don't hold anything alive.

Also, let's avoid lower-case "may" and observational statements in general in normative prose.

User agents generally need requirements NOT to save resources. Is this trying to say:
```suggestion
            for transport. The [= ICE agent =] MUST NOT free up candidates until all their associated candidate pairs have been removed.
```
?

Reply:
It's also unclear what "free up candidates" means. It seems to refer to the underlying concept "candidate" in [RFC8445](https://www.rfc-editor.org/rfc/rfc8445#section-2.1), not the [RTCIceCandidate](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate) WebIDL API object.

Do we need to say anything at all?

Reply:
Ah "free candidates" is https://datatracker.ietf.org/doc/html/rfc8445#section-8.3. We should probably clarify that with a link if we need this at all.

Reply:
Replaced the observational statement with the effects and the intended use of the method instead.